synbiohub_adapter/upload_sbol/upload_sbol.py

```diff
@@ -455,10 +492,10 @@ def query_collection_members(self, member_uris, collection_uris=[], rdf_type=Non
                     collection_uri = binding['collection']['value']
                 try:
                     collection_to_member[collection_uri].append(binding['entity']['value'])
-                except:
+                except KeyError:
                     try:
                         collection_to_member[collection_uri] = [binding['entity']['value']]
-                    except:
+                    except KeyError:
```

Comment:
Why is a KeyError ok to ignore in this context? A comment could be added to explain to a future programmer what the logic is. And under what circumstances a KeyError is expected or anticipated.

Reply:
This is likely a poor substitute of try/except for if statements. I rewrote this code.